javascript-light-bulb/main.js

```diff
@@ -0,0 +1,14 @@
+
+var $lightBulb = document.querySelector('.light-bulb-on');
+var $background = document.querySelector('.background-on');
+$lightBulb.addEventListener('click', lightSwitch);
+
+function lightSwitch(event) {
+  if (event.target.className === 'light-bulb-shape light-bulb-on') {
+    $lightBulb.className = 'light-bulb-shape light-bulb-off';
```

Comment:
Instead of checking the current class name, make a boolean variable to use as your data model and base your conditional logic on that.